xdg-terminal-exec.1

```diff
@@ -0,0 +1,63 @@
+.\"	Title : xdg-terminal-exec
+.\"	Author : Nathan Pratta Teodosio
+.\"	June, 2024
+.\"
+.TH XDG-TERMINAL-EXEC 1 "Xdg-terminal-exec manual"
+.SH NAME
+xdg-terminal-exec \- terminal execution utility
+.SH SYNOPSIS
+\fBxdg-terminal-exec [command [arguments]]
+.BR
+.SH DESCRIPTION
+Xdg-terminal-exec launches the preferred system terminal, as determined
+by the XDG specification.
+
+The preferred terminal is determined from the following config files, in
+order of decreasing priority:
+
+    ${HOME}/.config/${desktop}-xdg-terminals.list
+    ${HOME}/.config/xdg-terminals.list
+    /etc/xdg/${desktop}-xdg-terminals.list
+    /etc/xdg/xdg-terminals.list
+
+Which shall contain the desired terminals in the form of desktop entries
+as found in
```

Comment:
`${XDG_DATA_HOME}:${XDG_DATA_DIRS}` directory sequence deserves to be mentioned here.